specification/terms.md

```diff
@@ -70,28 +72,32 @@ A typed expression that evaluates to an object, as opposed to a value. The LHS o
 
 If a vexpr is used where an lexpr is expected (other than the LHS of an assignment, where using a vexpr is illegal), it is wrapped in an implicit temporary lexpr, the rules of which are given in §2.3.
 
-### §2.2 Value Expression (vexpr)
+### §2.2 Value Expressions (vexprs)
 
 A typed expression that evaluates to an value, as opposed to an object. The RHS of an assignment and the arguments of a function are examples of vexprs.
 
-If a lexpr is used where an vexpr is expected, the memory is read from the object referred to by the lexpr to create a vexpr *if* either T: Copy, or the mutability of the lexpr allows for moving.
-
-### §2.3 Temporary lexpr
-
-An implicit lexpr that wraps a vexpr when an lexpr is expected.
+NOTE: Because of evaluation order concerns, after being potentially wrapped to a
+vexpr, subexpressions are often wrapped in a `Use(Temp(vexpr))`. The
+exact rules here should be documented.
```

Comment:
Is this a Rust thing, or a rustc thing? i.e., should it be documented in the specification? Do we care that it gets put into a lexpr?